frb_example/pure_dart_multi/dart/lib/main.dart

```diff
@@ -0,0 +1,435 @@
+import 'dart:ffi';
+import 'dart:typed_data';
+
+import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
+import 'bridge_generated_api_1.dart';
+import 'bridge_generated_api_2.dart';
+import 'package:test/test.dart';
+
+void main(List<String> args) async {
+  String dylibPath1 = args[0];
+  String dylibPath2 = args[1];
+  print(
+      'flutter_rust_bridge example program start (dylibPath=$dylibPath1,$dylibPath2)');
+  print('construct api');
+  final dylib1 = DynamicLibrary.open(dylibPath1);
+  final api1 = ApiClass1Impl(dylib1);
```

Comment:
Btw are you using api2?

Reply:
I used, but at the very end of this function. Fixed in 3917234ba483bcd16ba702640a8d4b806d27d912.

Reply:
> Sure. Just fix the CI otherwise we will have a broken CI on master which is terrible

I don't know why it failed? What does `checkout@v2` refer to?

Reply:
> Use a yaml verifier to check it.

Now everything is ok.